.github/workflows/hub-deploy-sandbox.yaml

```diff
@@ -0,0 +1,42 @@
+name: Deploy-sandbox
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: sourcecode/hub
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          tools: composer:v2
+          coverage: none
+      
+      - name: Require Vapor CLI
+        run: composer global require laravel/vapor-cli
+
+      - name: Install Project Dependencies
+        run: composer install --no-interaction --prefer-dist --no-dev --optimize-autoloader
+
+      - name: Run PHP artisan
+        run: php artisan event:cache
+
+      - name: Install NPM
```

Comment:
This step is [redundant](https://docs.npmjs.com/cli/v9/commands/npm-ci) to `npm ci` below.

Reply:
Removed